frontend/packages/topology/src/components/utils/checkResourceQuota.tsx

```diff
@@ -0,0 +1,37 @@
+import { getUsedPercentage } from '@console/app/src/components/resource-quota/utils';
```

Comment:
should this file name be `checkResourceQuota.ts` ?

Reply:
Updated